front/src/applications/operationalStudies/views/Study.tsx

```diff
@@ -291,7 +277,7 @@ export default function Study() {
                   <div className="study-details-description">{study.description}</div>
                   {study.state && (
```

Comment:
This way, you can remove the ? (line 280)
```suggestion
                  {study.state && studyStates && (
```